docs/examples/README.md

```diff
@@ -0,0 +1,69 @@
+# Contributing Hydrogen examples
```

Comment:
This is great — but one thing I'm wondering is if Contributing should really be the H1, or potentially an H2. Most people coming to this folder will be consumers, instead of contributors, and I'm wondering if we should have a short part above this that explains/helps you navigate the examples.

Reply:
Yeah, we could even move this to its own `contributing.md` file and keep the main readme focused on end-users.

Reply:
Just pushed an update: the contribution readme stays where it is, inside `/docs`, but I've added a readme to `/examples` with some context for people browsing there, and linking out to other surfaces for contributing or requesting examples. Thoughts?